src/renderer/themes.css

```diff
@@ -127,6 +127,17 @@ it can be safely elided. This looks quite pleasant on this theme. */
 
 /*************** LIGHT THEME ADJUSTMENTS ***************/
 
+.system[data-system-theme*='light'],
+.light,
+.pastelPink,
+.everforestLightHard,
+.everforestLightMedium,
+.everforestLightLow,
+.gruvboxLight,
+.solarizedLight {
+  --primary-shadow-color: rgb(255 255 255 / 75%);
```

Comment:
> @efb4f5ff-1298-471a-8973-3d47447115dc The problem seems to be that this relies on the opacity of 75%; This does not work with the light theme because it is plain white? I am not sure on how to fix this. I mean I could make the light theme have its own shadow color (like it had before), but then this might be a issue which reappears when someone else creates a theme as well?

I'd recommend having one block for light themes that work better with a dark shadow and one block for themes that work better with a light, then add code comments above each block to clarify that.